proposals/Simple-programs.md

```diff
@@ -93,11 +89,6 @@ The method is designated as the entry point of the program. Explicitly declared
 could be considered as an entry point candidates are ignored. A warning is reported when that happens. It is
 an error to specify `-main:<type>` compiler switch.
 
-If any one compilation unit has statements other than local function declarations, statements from that
```

Comment:
>If [](start = 0, length = 2)

Deleting this paragraph (and related examples) doesn't make it illegal for a local function in one compilation unit to reference a local declared in another.  That is controlled by the scope, and your section "Scope of top-level local variables and local functions" below indeed makes that legal.  If you want it to be illegal, you should do so explicitly.  Otherwise it would be better to restore this text.

Reply:
> If you want it to be illegal, you should do so explicitly.

The following is added above: "Only one *compilation_unit* is allowed to have the *statement*s." I think that is explicit.

Reply:
It replaces the old rule in place: "In all but one *compilation_unit* the *statement*s must all be local function declarations."